e2e/tests/functional/forms.e2e.spec.js

```diff
@@ -91,6 +93,45 @@ test.describe('Persistence operations @addInit', () => {
     });
 });
 
+test.describe('Persistence operations @couchdb', () => {
+    test.use({ failOnConsoleError: false });
+    test('Editing object properties should generate a single persistence operation', async ({ page }) => {
+        test.info().annotations.push({
+            type: 'issue',
+            description: 'https://github.com/nasa/openmct/issues/5616'
+        });
+
+        await page.goto('./', { waitUntil: 'networkidle' });
+
+        // Create a new 'Clock' object with default settings
+        const clock = await createDomainObjectWithDefaults(page, {
+            name: `clock-${uuid()}`,
```

Comment:
I think we should consider making `objectname-uuid` the standard and include the `testInfo` api so that we can debug on banner.

`objectname-${uuid()}-${testInfo()}`

Reply:
We employ this pattern elsewhere, I think `objectname-${uuid()}` is a good approach. Including the test name or path might also be helpful, but we have a lot of options using the testInfo api

Reply:
https://github.com/nasa/openmct/issues/5513